src/App.js

```diff
@@ -14,11 +15,86 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGood: '',
+  }
+
+  addSelection = (event, item) => {
+    if (!event.ctrlKey) {
+      this.setState({ selectedGood: item });
+
+      return;
+    }
+
+    event.stopPropagation();
```

Comment:
why do you use it?

Reply:
i thought it was necessary for proper work